src/index.ts

```diff
@@ -168,20 +170,18 @@ function teenyRequest(reqOpts: r.Options, callback?: r.RequestCallback) {
             return;
           }
 
-          const encoding = res.headers.get('content-encoding');
```

Comment:
Can you explain why this is getting deleted? I just don't understand what it was doing is all :)

Reply:
Previously, the `response` event emitted `res.body`, and this adds a "headers" property to that object that only contains the `content-encoding` header.

As we found out, `res.body` is not the correct event message for `response`, so we emit an object that is similar to `http.IncomingMessage`, which is what the `request` module emits, with the properties `headers`, `statusMessage` and `statusCode`.

So I just take the response header from `node-fetch` and emit back out.